kinetic/distribution.yaml

```diff
@@ -8108,11 +8108,11 @@ repositories:
   multipoint_navigation_receiver:
     doc:
       type: git
-      url: https://github.com/SuperAruba/multipoint_navigation_receiver.git
+      url: https://github.com/Inspur-ROS/multipoint_navigation_receiver.git
```

Comment:
why is being the organization updated to a fork?

Reply:
This is our own organization for putting multiple codes together,and Fork is for easy maintenance

Reply:
Ok, considering that the same people seem to be the maintainers of both the fork and the original repo, this looks fine to me.